examples/resnet-18/resnet-18.py

```diff
@@ -0,0 +1,78 @@
+import numpy as np
```

Comment:
minor: A README.md would be extremely helpful to add for this

Reply:
Hi, thanks for the comments. I added a README.md with some instructions.